frontend/src/components/DatePicker/DateInput.tsx

```diff
@@ -65,6 +71,14 @@ export const DateInput = forwardRef<DateInputProps, 'input'>(
       [onChange],
     )
 
+    /**
+     * Disallow manual input of non-existant dates (e.g. 31 Sep 2022)
+     */
+    const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+      if (isValid(parse(e.target.value, 'yyyy-MM-dd', new Date())))
```

Comment:
i don't think this will work; this may prevent users from changing the dates entirely, or typing a date manually i think. Since the start of a date will always be invalid

Reply:
as discussed, date inputs behave a little differently... will probably need some changing later when the date input changes to a text one, but for now it's ok